calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_models/ceres_camera_model.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 TIER IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List
+from typing import Optional
+
+from intrinsic_camera_calibrator.camera_models.camera_model import CameraModel
+from intrinsic_camera_calibrator.camera_models.opencv_camera_model import OpenCVCameraModel
+import numpy as np
+
+os.environ["GLOG_minloglevel"] = "2"  # cSpell:ignore minloglevel
```

Comment:
Can you add this as an optional step linked to the verbosity of the calibrator?

Reply:
We do not handle verbosity via arguments. I can add `glog_level` parameter to config file, but setting environment variable still has to be done before importing Ceres library bindings. This force us to place `import` e.g. in class constructor, which is far away from being *pythonic*. Before I will do it, let me know how you see it because there is many places where we can do it, none of them are OK for me.
The best solution could be setting verbosity directly in our Ceres library, however glog ignores it for me and that's why I suppress optimization warnings by setting environment variable.

Reply:
Ok, I found this https://github.com/tier4/CalibrationTools/pull/208/commits/34679cb30708feb3e26624c318b5db30ba8f0809
For me this is acceptable